### DISCUSSION TITLE
Change query execution time (N rows fetched - Xs (Ys fetch) to always show in milliseconds

### DISCUSSION THREAD

**Discussion by CrimsonVex** [dbeaver/dbeaver#22632] [Q&A]
How can I change the the query execution time (bottom right corner status bar that reads: `N rows fetched - Xs (Ys fetch)`

I want it to always show in milliseconds. Sometimes for queries running in 1s - 2s, I'm curious ot know if it's 1100ms vs 1900ms. 
Can this be done?

**Comment by InfiniteEchoDev** [comment]
Seconded, would love this as well.

**Comment by LonwoLonwo** [comment]
You can create a feature request for this idea on our issue tracker.

**Comment by MilesTails01** [comment]
+1 definitely a feature that should be implemented

**Comment by CrimsonVex** [comment]
I've submitted a [feature request](https://github.com/dbeaver/dbeaver/issues/26658).

**Comment by E1izabeth** [comment]
You can see execution time in milliseconds in Query Manager?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/e4ca0ec8-863a-46f7-9712-4c8965913fe9)

You can open it here 
![image](https://github.com/dbeaver/dbeaver/assets/28875055/64e98d58-019b-4755-a2ef-eeed5a64c51e)

**Reply by MikhailGolubtcov** [reply]
Hi. @E1izabeth thanks for your help. I have seconds in my Query Manager, how to display milliseconds?

**Reply by E1izabeth** [reply]
@MikhailGolubtcov there is no way to do it, I created the issue about that https://github.com/dbeaver/dbeaver/issues/36742
